packages/remark-shaku-code-annotate/src/index.ts

```diff
@@ -227,6 +245,11 @@ function isCommentLine(line: IThemedToken[]) {
     )
   );
 }
+function hasFocusElement(line: IThemedToken[][]) {
+  return line.some((innerLine) =>
+    innerLine.some((element) => element.content.includes("@focus"))
```

Comment:
this is sketchy,  we've already parsed the lines,
so we can just check if there is line of DirectiveFocus?

Reply:
> this is sketchy, we've already parsed the lines, so we can just check if there is line of DirectiveFocus?

fixed.